### DISCUSSION TITLE
.echo vs .in

### DISCUSSION THREAD

**Discussion by acrlakshman** [sidbannet/converge_cfd_user#16] [General]
Hi @sidbannet, may I know if there is a reason why you are relying on .echo files rather than .in? Wouldn't it be more convenient to use .in which is in YAML format since v3.0 and can be parsed using available `yaml` package in python? If there is any reason that I missed, which motivated you to use .echo file, please let me know.

**Comment by sidbannet** [comment]
Good point. I debated with myself about this first when I was still developing for v2.4. Initially kept it simple with *.echo as they are the output of what code is seeing rather than the user input. There could be some difference depending on restarts and handling files. I think to your point, we can put in another class to get data from inputs (but only for >v3.0)

**Reply by acrlakshman** [reply]
Thanks for the input, I will start putting together something for this.
